Library/Navigation.swift

```diff
@@ -260,6 +267,33 @@ private let routes: [String:(RouteParams) -> Decoded<Navigation>] = [
   "/users/:user_param/surveys/:survey_response_id": userSurvey
 ]
 
+private let deepLinkRoutes: [String:(RouteParams) -> Decoded<Navigation>] = allRoutes.allValues(
```

Comment:
oh actually, using Hoogle I found this function:

```haskell
restrictKeys :: Ord k => Map k a -> Set k -> Map k a
```

which is a better name and a better definition! maybe we should even add this to the prelude...